src/component/App.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import Display from './Display'
+import ButtonPanel from './ButtonPanel'
+import calculate from '../logic/calculate'
+import './App.css'
+import { AppState, clickHandlerFunction } from './types'
+
+// export default class App extends React.Component {
+//   state = {
+//     total: null,
+//     next: null,
+//     // eslint-disable-next-line react/no-unused-state
+//     operation: null,
+//   }
+
+//   handleClick = (buttonName) => {
+//     console.log(this.state)
+//     this.setState((prevState) => calculate(prevState, buttonName))
+//   }
+
+//   render() {
+//     return (
+//       <div className="component-app">
+//         <Display value={this.state.next || this.state.total || '0'} />
+//         <ButtonPanel clickHandler={this.handleClick} />
+//       </div>
+//     )
+//   }
+// }
+
+export default function App() {
+  const [ownState, setOwnState] = useState<AppState>({
```

Comment:
> Is this a descriptive state variable name?
It isn't. At first it was as `state` and then I changed it to `ownState` because I was mixing the names of `useState` and `setState` functions. Changing it back to `state` now